libraries/rush-lib/src/logic/base/BaseInstallManager.ts

```diff
@@ -572,6 +572,14 @@ ${gitLfsHookHandling}
 
       if (experiments.usePnpmFrozenLockfileForRushInstall && !options.allowShrinkwrapUpdates) {
         args.push('--frozen-lockfile');
+
+        if (
+          options.pnpmFilterArguments.length > 0 &&
+          semver.satisfies(this.rushConfiguration.packageManagerToolVersion, '^8')
```

Comment:
Feels like we must already have the major version parsed out somewhere.

Reply:
Could just do a `.startsWith('8.')` I guess. I don't see it parsed anywhere here.